test/e2e/singlecluster/visibility_test.go

```diff
@@ -526,6 +532,14 @@ var _ = ginkgo.Describe("Kueue visibility server", func() {
 				},
 			}
 			gomega.Expect(k8sClient.Create(ctx, roleBinding)).Should(gomega.Succeed())
+			ginkgo.By("Returning a ResourceNotFound error for a nonexistent LocalQueue", func() {
```

Comment:
```suggestion
			ginkgo.By("Wait for ResourceNotFound error instead of Forbidden to make sure the role bindings work", func() {
```
nit, and use the same for the other block

Reply:
Or even better, wrap in a function `awaitForInitialization`, and call from both `BeforeEach`

Reply:
> Or even better, wrap in a function `awaitForInitialization`, and call from both `BeforeEach`

Unfortunately they are different.

Reply:
ah, I see, thanks, this binding in the second `BeforeEach` is for nsA